main.go

```diff
@@ -21,14 +22,26 @@ func strAddr(s string) uint32 {
 	return ret
 }
 
+var (
+	flagMgmtListen string
```

Comment:
I prefer the `flag.XVar` over `flag.X` convention because then you end up with plain values, instead of pointers to them.

I can switch to the other if you care much for that.

Reply:
According to https://godoc.org/flag#String we would still have pointers.